lib/Dialect/FIRRTL/Transforms/BlackBoxReader.cpp

```diff
@@ -376,7 +399,7 @@ bool BlackBoxReaderPass::isDut(Operation *module) {
   if (node)
     for (auto *u : node->uses()) {
       // Recursively check the parents.
-      auto dut = isDut(u->getInstance()->getParentOfType<FModuleOp>());
+      auto dut = isDutOrNoDut(u->getInstance()->getParentOfType<FModuleOp>());
```

Comment:
Maybe changing the `dut` to something like `isDut` makes sense, since it conflicts with the   `FModuleOp dut;`.

Reply:
Good point. I think the cleanest change would be to add logic into `isDut` that makes it `shouldBeInTestBenchDir`. This then avoids needing to pass `dut` information into the function and makes it more obvious what the purpose of the function is.

Reply:
Actually, it's better to just keep the existing `isDut` method and then just do the gating that the `dut` exists inside the `setOutputFile` member function. I'll rework this to remove the `bool isDut` in a later commit.